agent/host_agent_test.go

```diff
@@ -545,7 +578,7 @@ var _ = Describe("Agent", func() {
 					}
 				}
 				return false
-			}).Should(BeTrue())
+			}, time.Second*2).Should(BeTrue())
 		})
 
 		It("should not register the BYOHost with the management cluster", func() {
```

Comment:
Is this assertion still relevant?

Reply:
couldn't get it. Could you elaborate a bit?